diktat-ruleset/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSaveSmokeTest.kt

```diff
@@ -32,13 +32,17 @@ import kotlin.system.measureNanoTime
 
 @DisabledOnOs(OS.MAC)
 class DiktatSaveSmokeTest : DiktatSmokeTestBase() {
-    override val isLintErrors = false
     override fun fixAndCompare(
-        config: String,
+        config: Path,
         expected: String,
         test: String,
+        trimLastEmptyLine: Boolean,
     ) {
-        saveSmokeTest(Path(config), test)
+        saveSmokeTest(config, test)
+    }
+
+    override fun assertUnfixedLintErrors(lintErrorsConsumer: (List<LintError>) -> Unit) {
```

Comment:
Up to you, but the empty function body here can be replaced with

```kotlin
    /***
     * Does nothing, we can't check unfixed lint errors here.
     */
    override fun assertUnfixedLintErrors(lintErrorsConsumer: (List<LintError>) -> Unit) = Unit
```

I personally think this is more idiomatic, but I may be biased.